unit_tests/UnitTestOutputManager.C

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+#include "OutputInfo.h"
+#include "gtest/gtest.h"
+
+#include "NaluParsing.h"
+
+namespace sierra {
+namespace nalu {
+namespace {
+TEST(OutputManagerTest, can_read_multiple_output_nodes)
```

Comment:
Can you also add a unit test to write multiple output ?

Reply:
@tokusanya can you point me to a good example of how to do this? is there a way to call the write operation without actually having to generate the files?

Reply:
@psakievich I don't know how nalu represents meshes or if you have some simple fixture that allows you to create a small mesh with multiple data that you would want to output to different output streams but STK unit tests typically create small easily managed meshes that it can manipulate for testing.

Reply:
Here is the mesh creation function used by unit tests: 

https://github.com/Exawind/nalu-wind/blob/154f167f063c68096bf5f55152fa5b0b4c568993/unit_tests/UnitTestUtils.C#L74-L88